.github/workflows/build.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         plugins: [false]
         skip-artifact: [false]
         skip-crashpad: [false]
+        skip-pch: [false]
```

Comment:
I'd rather this be `enable-pch` with the default value being `false` since we only want PCH enabled for 2 of the many jobs